### DISCUSSION TITLE
Historical chat records

### DISCUSSION THREAD

**Discussion by Danrry1996** [infiniflow/ragflow#2416] [Q&A]
Why is it possible to save users' chat history in locally deployed RagFlow chat, while the web does not have this feature?
![image](https://github.com/user-attachments/assets/b550f351-1105-4017-8fed-ac6359b06b4c)

![image](https://github.com/user-attachments/assets/cca9f369-35da-4cc9-8682-071fb4e36672)

**Comment by KevinHuSh** [comment]
You could file a [Feature Request].
